docs/dev/news/5281.feat

```diff
@@ -0,0 +1 @@
+Improved error reporting when verification times out or runs out of resources, so that when using --isolate-assertions, the error message points to the problematic assertion.
```

Comment:
```suggestion
Improved error reporting when verification times out or runs out of resources, so that when using `--isolate-assertions`, the error message points to the problematic assertion.
```